.travis.yml

```diff
@@ -40,6 +40,21 @@ script:
 - bundle exec rake puppetserver
 after_script:
 - docker-compose -f spec/docker-compose.yml logs
+deploy:
+  provider: script
+  on:
+    # deploy the master branch or tag pushes
+    condition: $TRAVIS_BRANCH =~ ^master|\d+\.\d+\.\d+$
```

Comment:
I think
```
on:
    repo: puppetlabs/bolt
    branch: master
```
may be sufficient. I don't think TRAVIS_BRANCH will be set to a tag name if the tag is pushed.

Reply:
https://github.com/travis-ci/travis-ci/issues/7780 has some discussion of ways to do master and tags.

Reply:
Somewhere else in discussion of how to do tags it suggested that TRAVIS_BRANCH is set to the tag.